tensorflow/contrib/receptive_field/python/util/receptive_field.py

```diff
@@ -331,6 +394,18 @@ def compute_receptive_field_from_graph_def(graph_def, input_node, output_node):
       cannot be found. For network criterion alignment, see
       photos/vision/features/delf/g3doc/rf_computation.md
   """
+  # Convert a graph to graph_def if necessary
```

Comment:
It seems like the function now supports graph_def to be a simple graph, and input and output nodes to be tensors. Please add a few comments to the docstring to reflect these changes.